docs/UserGuide.md

```diff
@@ -148,6 +148,20 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Adding a course: `course-add`
+
+Adds a course to TAHub Contacts.
+
+Format: `course-add c/COURSE_CODE n/COURSE_NAME`
+
+Examples:
+* course-add c/CS1101S n/Programming Methodology 1
```

Comment:
The User Guide on Google Docs uses the ```add-course``` command though? Not sure if this was a mistake or the command was changed?